modules/kmm-configuring-the-lookup-path-on-nodes.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-configuring-the-lookup-path-on-nodes_{context}"]
+= Configuring the lookup path on nodes
+
+On OpenShift nodes, the set of default lookup paths for firmwares does not include `/var/lib/firmware`.
+
+.Procedure
+
+. Use the Machine Config Operator to create a `MachineConfig` resource that contains the `/var/lib/firmware` path:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-kernel-args-firmware-path
+spec:
+  kernelArguments:
+    - 'firmware_class.path=/var/lib/firmware'
+----
+
+. Reboot all of the worker nodes.
```

Comment:
By applying the MachineConfig, the nodes will get automatically rebooted

Reply:
Added this to the step.